src/EditorFeatures/Core/Implementation/Structure/AbstractStructureTaggerProvider.cs

```diff
@@ -230,7 +233,7 @@ protected sealed override ITaggerEventSource CreateEventSource(ITextView textVie
 
             // Make sure the regions are lexicographically sorted.  This is needed
             // so we can appropriately parent them for BlockTags.
-            multiLineRegions.Sort((s1, s2) => s1.TextSpan.Start - s2.TextSpan.Start);
+            multiLineRegions.Sort(s_blockSpanComparer);
```

Comment:
You tell me.  The commit message was "work around bug"  What bug?  How do we know when to simplify again.